geometry_msgs/PoseStamped[] poses
int32[] tag_ids